partiql-lang/build.gradle.kts

```
@@ -0,0 +1,57 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+plugins {
+    id(Plugins.conventions)
+    id(Plugins.publish)
+}
+
+dependencies {
+    api(project(":partiql-ast"))
+    api(project(":partiql-eval"))
+    api(project(":partiql-parser"))
```

```suggestion
    api(project(":partiql-parser", configuration = "shadow"))
```
Do you need to specify the shadow configuration here? I haven't pulled this down.